src/crates-and-source-files.md

```diff
@@ -4,6 +4,7 @@ r[crate]
 r[crate.syntax]
 ```grammar,items
 @root Crate ->
+    FRONTMATTER?
     InnerAttribute*
     Item*
```

Comment:
Despite shebang's not having being here, I assumed I should put FRONTMATTER here

Reply:
The reason shebang isn't there is that shebangs are ignored at the start of any file, not just in the file which defines a crate.

If that's true for frontmatter too, `input-format.md` might be a better place to say how `frontmatter.md` fits in.